partitioning_expert-20171011.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tags": [
+    "really-delete"

```

Comment:
the file name should have this included. Also I recommend to be a bit more explicit here with the tag name, e.g. "partitioning_expert-really-delete". Then the filename could be correspondingly "partitioning_expert-really-delete-20171011.json"